### PR TITLE
update messaging

### PR DESCRIPTION
**Pull request by chrifro**
## Related Issue(s)
#3026

## Context
Remove student plans from our messaging [see internal discussion](https://gitpod.slack.com/archives/C01LDA27WJC/p1668107832961349)

## Outstanding to-dos:

- [ ] remove [GitHub Student page](https://www.gitpod.io/github-student-developer-pack)
- [ ] make sure the student plan pop-up on /pricing is fully removed
- [ ] double-check for any other student mentions
- [ ] see comments inline

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3027"></a>

### CODE REVIEW

File: src/lib/contents/pricing.ts [hunk 1]
```diff
@@ -534,6 +517,12 @@ export const pricingFAQ: FAQ = {
       content:
         "<p> If you wish to stop using Gitpod, you may cancel your subscription at any time. Your cancellation will take effect after that month’s billing cycle. </p>",
     },
+    {
+      title:
+        "Do you offer discounts for students and educational institutions?",
+      content:
+        "<p> Students are welcome to take advantage of our generous generous Free and Personal plans. </p>",
```
Comment (chrifro) [inline src/lib/contents/pricing.ts:524]: Do we want to keep it like this or remove the whole questions?
cc @MichaelAring

File: src/lib/contents/education.ts [hunk 2]
```diff
@@ -32,14 +32,6 @@ export const features: Feature[] = [
 ];
 
 export const programBenefits: Card[] = [
-  {
-    icon: {
-      src: "/svg/media-kit/logo-mark.svg",
-      alt: "Gitpod",
-    },
-    title: "75% student discount",
```
Comment (chrifro) [inline src/lib/contents/education.ts:40]: Should we delete the whole `"program benefit"` section? Just mentioning `"leading technology` and `"save time" `might seem like "weak" arguments. 
cc @MichaelAring

File: src/lib/contents/pricing.ts (hunk 1 above)
Comment (MichaelAring) [reply src/lib/contents/pricing.ts:524]: I would remove the whole question and only add it back if we hear this question repeatedly in the community, contact form or elsewhere.

**Comment by jldec** [comment]
This looks good - thanks @chrifro and @MichaelAring 🙏

**Review by MichaelAring — APPROVED** [review]

**Pull request merged**